packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+            height: 74px;
+            border-top: 1px solid #F0F0F1;
+        }
+
+    }
+
+    .confirm-txt {
+        font-size: 14px;
+        line-height: 150%;
+        color: #72727A;
+        margin-left: 16px !important;
+    }
+
+
+    .h-right {
+        position: absolute;
+        right: 16px;
+        text-align: right;
+    }
+
+    .estimate-fee-card {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+
+        width: 100%;
+        height: 78px;
+        margin-top: 16px;
+
+        border: 1px solid #F0F0F1;
+        box-sizing: border-box;
+        border-radius: 8px;
+    }
+
+    .account-id {
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 150%;
+        text-align: right;
+        color: #272729;
+    }
+
+    h3 {
+        font-weight: 900;
+        font-size: 20px;
+        line-height: 130%;
+        text-align: center;
+        color: #24272A;
+    }
+
+    .success {
+        > p {
+            font-weight: 500;
+            font-size: 20px;
+            line-height: 150%;
+            text-align: center;
+            color: #3F4045;
+
+            margin-bottom: 0px;
+        }
+    }
+
+    p {
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 150%;
+        text-align: center;
+        color: #72727A;
+    }
+
+    .next-btn {
+        margin-left: 44px !important;
+    }
+
+    .receiver-input {
+      width: 100%;
+    }
+
+    .amount-grey {
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 150%;
+        color: #A2A2A8;
+    }
+
+    .v-center {
+        position: absolute;
+        top: 50%;
+        transform: translate(0, -50%);
+    }
+
+    .icon {
+        margin-bottom: 20px !important;
+    }
+
+    form {
+        width: 100%;
+    }
+
+    .full-width {
+        width: 100%;
+    }
+
+
+    .modal-footer {
+        display: flex;
+        align-items: right;
+        justify-content: flex-end;
+
+        button {
+            width: 136px !important;
+            height: 56px !important;
+            margin-top: 0px !important;
+
+            &.link {
+                margin: 20px 35px;
+            }
+            
+            &.blue {
+                padding: 0 35px;
+            }
+        }
+    }
+
+    .success-bottons {
+        align-items: center;
+        margin-left: auto;
+        margin-right: auto;
+
+        > button {
+            width: 185px !important;
+        }
+    }
+`;
+
+export default function NFTTransferModal({ open, onClose, nft, accountId, setOwnerId }) {
+    const [ receiverId, setReceiverId ] = useState('');
+    const [ result, setResult ] = useState();
+    const [ sending, setSending ] = useState(false);
+    const [ viewType, setViewType ] = useState('transfer');
+    const [ accountIdIsValid, setAccountIdIsValid] = useState(false);
+    const fungibleTokens = useFungibleTokensIncludingNEAR();
+    const nearBalance = fungibleTokens[0].balance;
```

Comment:
Let's use the selector here too: https://github.com/near/near-wallet/pull/2205/files#r758852017